cabot/cabotapp/migrations/0019_auto__add_schedule__add_field_shift_schedule__del_field_userprofile_fa.py

```diff
@@ -0,0 +1,235 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding model 'Schedule'
+        db.create_table(u'cabotapp_schedule', (
+            (u'id', self.gf('django.db.models.fields.AutoField')(primary_key=True)),
+            ('name', self.gf('django.db.models.fields.TextField')(unique=True)),
+            ('ical_url', self.gf('django.db.models.fields.TextField')()),
+            ('fallback_officer', self.gf('django.db.models.fields.related.ForeignKey')(to=orm['auth.User'], null=True, blank=True)),
+        ))
+        db.send_create_signal(u'cabotapp', ['Schedule'])
+
+        # Adding field 'Shift.schedule'
+        db.add_column(u'cabotapp_shift', 'schedule',
+                      self.gf('django.db.models.fields.related.ForeignKey')(default=1, to=orm['cabotapp.Schedule']),
+                      keep_default=False)
+
+        # Deleting field 'UserProfile.fallback_alert_user'
+        db.delete_column(u'cabotapp_userprofile', 'fallback_alert_user')
```

Comment:
Yeah I didn't see any reason to keep fallback_officer as a field for user...but let me know if you think of anything